.nvmrc

```diff
@@ -0,0 +1 @@
+lts/*
```

Comment:
is this right?

Reply:
yes: https://github.com/creationix/nvm#nvmrc